File: PHP/UberBlack.php
<?php

require_once('./Car.php');

class UberBlack extends Car
{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial)
    {
        parent::__construct($license, $driver); #Equivale al super de Java
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }

    
}


?>

File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    
    typeCarAccepted = []
    seatsMaterials = []

    def __init__(self, licence, driver, typeCarAccepted, seatsMaterials):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterials = seatsMaterials